plugins/integration/providers/voicebase/lib/KVoicebaseIntegrationEngine.class.php

```diff
@@ -69,6 +70,16 @@ protected function doDispatch(KalturaBatchJob $job, KalturaIntegrationJobData &$
 	
 	protected function doClose(KalturaBatchJob $job, KalturaIntegrationJobData &$data, KalturaVoicebaseJobProviderData $providerData)
 	{
+		$entryId = $providerData->entryId;
+		$this->clientHelper = VoicebasePlugin::getClientHelper($providerData->apiKey, $providerData->apiPassword);
+		$remoteProcess = $this->clientHelper->retrieveRemoteProcess($entryId);
+		
+		//false result means that something has gone wrong - the VB job is either in status error or missing altogether
+		if(!$remoteProcess || $remoteProcess->requestStatus == VoicebaseClientHelper::VOICEBASE_FAILURE_MEESAGE || !isset($remoteProcess->fileStatus) || $curlResult->fileStatus == VoicebaseClientHelper::VOICE_MACHINE_FAILURE_MESSAGE)
```

Comment:
Do we know for sure that `$remoteProcess->fileStatus` will always be set?
Because it currently checks for `!isset($remoteProcess->fileStatus)` and then assumes that there's an error

Reply:
@rkreich it has not happened in my testing. After calling mediaUpload for the entry ID, you have a file in "processing" status. During testing, I never had a job fail after upload (unless the video format was impossible for VB to handle (i.e .arf)